storehaus-elasticsearch/src/main/scala/com/twitter/storehaus/elasticsearch/ElasticSearchStringStore.scala

```diff
@@ -70,10 +66,10 @@ class ElasticSearchStringStore(private val index: String,
         val response = request.execute().actionGet()
 
         response.iterator().asScala
-          .filter(r=>Option(r.getResponse)!=None)
+          .filter(r => Option(r.getResponse).isDefined)
```

Comment:
ahh, sorry. In library code, I advocate for a little bit of geeking out in the name of perf. In this case, actually `Option[T]` is not `TraversableOnce` but there is an implicit conversion. Since you are making the `Option` you could just have well have made the `Iterator` which is what this code wants (note the `.iterator().asScala` above).

`Iterator.single` is the cheapest way to make an iterator of one item, and likewise for empty.